common/src/main/java/com/microsoft/identity/common/internal/commands/RemoveCurrentAccountCommand.java

```diff
@@ -54,7 +56,24 @@ public RemoveCurrentAccountCommand(@NonNull RemoveAccountCommandParameters param
 
     @Override
     public Boolean execute() throws Exception {
-        return getDefaultController().removeCurrentAccount((RemoveAccountCommandParameters) getParameters());
+        final String methodName = ":execute";
+
+        for (int ii = 0; ii < getControllers().size(); ii++) {
```

Comment:
Minor code golf: could use a foreach if you don't need the index's value

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/961/commits/a683e653c33ce9a961675e44b4177a8e130ec589